### DISCUSSION TITLE
Do you plan on making the calendars synchronize automatically?

### DISCUSSION THREAD

**Discussion by Florian-Hink** [eoleedi/timetree-exporter#35] [General]
Hi eoleedi, 

thanks for digging into this topic!! It's a dick move from TimeTree to make it impossible to share the calendar with third party calendars. You help many people with your project.

One question: Do you plan on making the calendars synchronize automatically?

All the best,

Olf-123

**Comment by eoleedi** [comment]
@Olf-123  The short answer is no.

Synchronizing between TimeTree Calendar and other calendar services requires web scraping and polling manually, requiring a backend to update periodically. Since TimeTree Inc. terminates its API services, It becomes harder for me to fetch or update the data through an automatic pipeline reliably(Those Web APIs might change suddenly). 

Put that aside, I welcome any thoughts that may make that feature easier. Also, I'd like to know whether many people need to synchronize data between different calendars.
